clarifai/versions.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-CLIENT_VERSION = "10.5.2"
+CLIENT_VERSION = "10.5.3"
```

Comment:
maybe this should read `VERSION` instead of hardcoding?

Reply:
yeah I find this weird too. maybe @sanjaychelliah can take up